### DISCUSSION TITLE
User display name

### DISCUSSION THREAD

**Discussion by Sunderw3k** [serenity-rs/serenity#2500] [Q&A] [answered]
Is there a way to get the user display name from the `User` object? Using `name` gives back the handle.

**Comment by Sunderw3k** [comment]
it seems like the [user object](https://discord.com/developers/docs/resources/user#user-object) has `global_name` but that isn't being deserialized by the library

**Reply by tazz4843** [reply]
It is on next: https://serenity-rs.github.io/serenity/next/serenity/model/guild/struct.Member.html#method.display_name

**Reply by Sunderw3k** [reply]
That's the guild nickname tho, not exactly what I want. Also it wouldn't work for DMs

**Reply by arqunis** [reply, marked as answer]
> Also it wouldn't work for DMs

It would, actually, [if you look at the implementation](https://serenity-rs.github.io/serenity/next/src/serenity/model/guild/member.rs.html#230). If the user has it, it falls back to the [`User::global_name`](https://serenity-rs.github.io/serenity/next/serenity/model/user/struct.User.html#structfield.global_name) field, which is what you want.
